app/controllers/turfs_controller.rb

```diff
@@ -0,0 +1,23 @@
+class TurfsController < ApplicationController
+  def create
+    unless current_user
+      flash[:notice] = "You must be signed in to do that."
+      render "/"
+      return
+    end
+    params["lists"].each_value do |list|
+      @list = List.new
+      @list.name = list["name"]
+      if @list.save
+        list["voter_ids"].each do |voter_id|
+          ListMembership.create(voter_id: voter_id, list_id: @list.id)
```

Comment:
what happens if the `ListMembership` is invalid? Consider `create!` or managing the use case where a ListMembership would be invalid